src/main/scala/chisel3/iotesters/ChiselMain.scala

```diff
@@ -146,7 +146,7 @@ object chiselMain {
       case "firrtl" =>
         val file = new java.io.File(context.targetDir, s"${dut.name}.ir")
         val ir = io.Source.fromFile(file).getLines mkString "\n"
-        new FirrtlTerpBackend(dut, ir, context.testerSeed)
+        new FirrtlTerpBackend(dut, ir)
```

Comment:
--test-seed <long-value> will set it for all backends
The Backend(seed) is not really used but I found it to hard to remove it without breaking the public api for testers